src/main/scala/com/tapad/docker/DockerComposePlugin.scala

```diff
@@ -42,10 +42,12 @@ case class ServiceInfo(serviceName: String, imageName: String, imageSource: Stri
  *                          with an SBT project.
  * @param composeFilePath The path to the Docker Compose file used by this instance
  * @param servicesInfo The collection of ServiceInfo objects that define this instance
+ * @param variables A collection of key value pairs used for docker-compose variable substitution
  * @param instanceData An optional parameter to specify additional information about the instance
  */
 case class RunningInstanceInfo(instanceName: String, composeServiceName: String, composeFilePath: String,
-  servicesInfo: Iterable[ServiceInfo], instanceData: Option[Any] = None)
+  servicesInfo: Iterable[ServiceInfo], variables: Vector[(String, String)] = Vector.empty,
```

Comment:
You should now be able to remove the previously added `variables` property and comment from `RunningInstanceInfo`.